myft/templates/follow-collection.js

```diff
@@ -36,18 +36,19 @@ class FollowCollection extends Component {
 			action="#">
 			<input type="hidden" value={this.props.concepts.map(c => c.taxonomy).join(',')} name="taxonomy" />
 			<input type="hidden" value={this.props.concepts.map(c => c.name).join(',')} name="name" />
-			<button
-				aria-label={'Follow all topics in the ' + this.props.name + ' collection'}
-				aria-pressed='false'
-				className={buttonClasses.join(' ')}
-				data-alternate-label={'Unfollow all topics in the ' + this.props.name + ' collection'}
-				data-alternate-text="Following"
-				data-trackable="follow"
-				data-concept-id={conceptIds} // duplicated here for tracking
-					title={'Follow all topics in the ' + this.props.name + ' collection'}
-				type="submit">
-					Follow all
-			</button>
+			{(this.props.flags.myFtApiWrite) ? (
+				<button
+					aria-label={'Add all topics in the ' + this.props.name + ' collection to MyFT'}
+					aria-pressed='false'
+					className={buttonClasses.join(' ')}
+					data-alternate-label={'Remove all topics in the ' + this.props.name + ' collection for MyFT'}
```

Comment:
Also, just remembered, to get screen readers to pronounce myFT properly, we need to space out the letters like this: `my F T` (e.g. https://github.com/Financial-Times/next-myft-page/blob/b0d5b22ad02eda7831e4a93fd1be3197d9e2b098/views/partials/header.html#L4)